cli/cli.py

```diff
@@ -337,6 +357,12 @@ def check_error_option_presence(task: str | None, error: str | None) -> None:
         sys.exit(1)
 
 
+def check_error_measure_option_presence(task: str | None, error_measure: str | None) -> None:
```

Comment:
It seems, that we also should update function `check_error_option_presence`:
```py
if task in (Task.afd, Task.pfd) and error is None:
        click.echo(f"ERROR: Missing option '{ERROR}'.")
        sys.exit(1)
```
Because if we want find pfds, then we should also set error threshold.